src/Lykke.Quintessence.Core/Lykke.Quintessence.Core.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <RootNamespace>Lykke.Quintessence.Core</RootNamespace>
+        <Version>1.0.1</Version>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Lykke.Quintessence.Core.Abstractions\Lykke.Quintessence.Core.Abstractions.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Autofac" Version="4.8.1" />
+      <PackageReference Include="Multiformats.Hash" Version="1.4.0" />
+      <PackageReference Include="Nethereum.Signer" Version="3.0.0" />
+      <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
+    </ItemGroup>
+
```

Comment:
Make sure you ship pdb with nuget:
![image](https://user-images.githubusercontent.com/14955144/52265278-2286f000-294d-11e9-8f1f-6ad37cb68c93.png)